src/System.Runtime.Extensions/tests/System/Environment.TickCount.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Runtime.Extensions.Tests
+{
+    public class EnvironmentTickCount
+    {
+        [Fact]
+        public void TickCountTest()
+        {
+            //arrange
+            const int milliSeconds = 1000;
+            int start = Environment.TickCount;
+
+            //act
+            Task.Delay(milliSeconds).Wait();
```

Comment:
We've been trying hard to avoid artificial delays like this, especially in inner loop tests.  At the very least please shorten the delay significantly, e.g. 10 milliseconds rather than a second.